docs/releases/1.8.1.txt

```diff
@@ -11,3 +11,6 @@ Bugfixes
 
 * Added support for serializing :class:`~datetime.timedelta` objects in
   migrations (:ticket:`24566`).
+
+* Prevented ``TypeError`` in translation functions ``check_for_language`` and
```

Comment:
include () after function names for consistency with the output generated when using `:func:`